tests/framework/integration/installation_and_upgrade/run_tests

```diff
@@ -149,151 +120,72 @@ wait_for_nodes "$CLIENT_1" "rpm -q kmod-lustre-client || exit 0
 echo "Create and exercise a filesystem..."
 
 TESTS="$TESTS_DIR/../shared_storage_configuration/test_cluster_setup.py \
-       $TESTS_DIR/test_update_with_yum.py:TestYumUpdate.test_clean_linux_devices \
        $TESTS_DIR/test_create_filesystem.py:TestCreateFilesystem.test_create"
 
 # shellcheck disable=SC2086
 ssh root@$TEST_RUNNER "exec 2>&1; set -xe
 cd /usr/share/chroma-manager/
-unset http_proxy; unset https_proxy
-export UPGRADE_FROM_3=$UPGRADE_FROM_3
+export IML_4_INSTALLED=yup
```

Comment:
Why not "yes"? Just curious.

Reply:
It doesn't matter what's here, the test only checks for var existence.

Reply:
ok sounds good